drivers/wifi/simplelink/simplelink_log.h

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
I don't remember this file in the previous PR version ;-). Let me approve, but IMHO, it's superfluous.

Reply:
2 lines of defines specifically. It does so using 23 lines. And it takes 1 line to include it. And it's included 3 times. For me, the math doesn't even out, but that's just me ;-).